alicloud/resource_alicloud_ccn_instance.go

```diff
@@ -0,0 +1,185 @@
+package alicloud
+
+import (
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/requests"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/smartag"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-alicloud/alicloud/connectivity"
+)
+
+func resourceAlicloudCcnInstance() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAlicloudCcnInstanceCreate,
+		Read:   resourceAlicloudCcnInstanceRead,
+		Update: resourceAlicloudCcnInstanceUpdate,
+		Delete: resourceAlicloudCcnInstanceDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validateInstanceName,
+			},
+			"description": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validateInstanceDescription,
+			},
+			"is_default": {
+				Type:     schema.TypeBool,
+				Required: true,
+			},
+			"cidr_block": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validateVpnCIDRNetworkAddress,
+			},
+			"total_count": {
```

Comment:
I think there is no need to add  an extra parameter `total_count`. If you want to attach ccn and cen, please use a new resource `alicloud_cloud_connect_network_attachment`.

In addition, I noticed the schema parameters missing `SnatCidrBlock`, why did not add it?